bindings/2.206/Extras.bro

```diff
@@ -8,20 +8,20 @@ class AdvancedFollowInstance {
 }
 
 class SFXTriggerInstance {
-	PAD = win 0x10;
+	PAD = win 0x10, m1 0x0;
 	SFXTriggerGameObject* m_sfxTriggerGameObject;
 }
 
 class EventTriggerInstance {
-	PAD = win 0x10;
+	PAD = win 0x10, m1 0x0;
```

Comment:
yeah please do not use pad 0x0 anywhere that just defeats the purpose and it doesnt even work

Reply:
the thing is if i dont put them there the members are privated

Reply:
then put in the correct pads

Reply:
that's the problem, i can't see any